tests/data_types_test.ts

```diff
@@ -922,7 +922,7 @@ Deno.test(
 Deno.test(
   "date",
   testClient(async (client) => {
-    const date_text = "2020-01-01";
+    const date_text = "2020-01-02";
```

Comment:
Lol why

Not that I'm opposing just want to get the joke if there is any

Reply:
Oh, right... so this is related to timezones. The test break if you are in a UTC-n (instead of UTC+n) timezone :-P And I am in UTC-1 :-)

I don't actually want to merge this. There is a bug in the implementation here. I'll open a separate issue to get this fixed. You can probably reproduce by running the tests after setting `TZ=Europe/Amsterdam`.

Reply:
Oh damn, I hadn't even realized this

I'll submit a fix right now